scripts/run-ci-tests.sh

```diff
@@ -77,11 +77,11 @@ else
                 ;;
             *continuous*)
                 TEST_CONFIG="Continuous"
-		CTEST_OPTIONS+=" --timeout 360"
+		CTEST_OPTIONS+=" --timeout 480"
```

Comment:
do the new Xcode tests take longer?

Reply:
It seems like they might. The Application API tests took 470 seconds for one of the runs that passed. I noticed one of the Linux tests taking longer than the previous timeout.